aws-resources/src/main/java/io/opentelemetry/contrib/aws/resource/EksResource.java

```diff
@@ -72,6 +73,9 @@ static Resource buildResource(
     String clusterName = getClusterName(httpClient);
     if (clusterName != null && !clusterName.isEmpty()) {
       attrBuilders.put(ResourceAttributes.K8S_CLUSTER_NAME, clusterName);
+
+      String logGroupName = String.format("/aws/containerinsights/%s/application", clusterName);
```

Comment:
Hello, does this mean there must be a group name literally ends with `application` and the full name must be `"/aws/containerinsights/{clusterName}/application"`? 

Just wondering / confirming because I do not see there is such a log group name in my environment.

If that's true, then we'll need to adjust our log group setup.

Reply:
This is good to know @BT-22 - this log group name is based on the log groups if you've setup CloudWatch Container Insights in your EKS cluster: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs-FluentBit.html#Container-Insights-FluentBit-setup

Do you know of anywhere in your EKS environment where you can inspect the name of the CW Log group you're sending log data to? A configuration file for instance?

Reply:
Thanks for the swift reply @willarmiros - while I'm still finding more details in my environment, I learned the following first:

> we are using a gruntworks module that wraps the fluent bit helm chart https://github.com/aws/eks-charts/tree/master/stable/aws-for-fluent-bit
> there's only one log group with that

Where I get the container logs is `/aws/eks/{clusterName}/container-logs`

Reply:
I see! In the repo you linked, it says you configure the log group destination: https://github.com/aws/eks-charts/tree/master/stable/aws-for-fluent-bit#configuration

So I wonder if we can read that config from the resource detector somehow...

Reply:
Yeah I do not think you can assume everyone uses the same log group name. Depending on how people get their container logs to cloudwatch. We like to have ours formatted a specific way, so I think this needs to be exposed in your collector so you can specify what it is. Ours is "/aws/eks/<eks clustername}/container-logs" as well. As BT-22 says, we are using fluent bit, but we pass the log group to a terraform module(gruntworks) that intern calls the fluent bit helm chart. I think this needs to be exposed in the helm charts for rolling out the collectors that collect this stuff and send it to aws?

Reply:
We actually use the adot operator and the adot collector statefulset to collect our traces and send to xray https://github.com/aws-observability/aws-otel-collector.